src/main/java/bio/terra/service/filedata/flight/ingest/FileIngestBulkFlight.java

```diff
@@ -173,6 +173,9 @@ public FileIngestBulkFlight(FlightMap inputParameters, Object applicationContext
             randomBackoffRetry);
         addStep(new IngestFileMakeBucketLinkStep(datasetBucketDao, dataset), randomBackoffRetry);
       }
+      addStep(
+          new ValidateBucketAccessStep(gcsPdao, userReq, dataset),
```

Comment:
This now runs for all GCP bulk file ingests - array or JSON, bulk mode or not.

For array ingests (bulk or not), this step validates accessibility of the files that the user specified directly in the ingest request.

For JSON ingests, this step validates accessibility of the control file.  The files to ingest have their accessibility validated downstream.